src/main/java/br/com/caelum/comidinhas/handler/validation/ValidationErrorsOutputDto.java

```diff
@@ -0,0 +1,30 @@
+package br.com.caelum.comidinhas.handler.validation;
+
+import java.util.*;
+
+public class ValidationErrorsOutputDto {
+
+    private List<String> globalErrorMessages = new ArrayList<>();
+    private List<FieldErrorOutputDto> fieldErrors = new ArrayList<>();
```

Comment:
A classe não precisa ser pública e os atributos podem ser final.

```suggestion
class ValidationErrorsOutputDto {

    private final List<String> globalErrorMessages = new ArrayList<>();
    private final List<FieldErrorOutputDto> fieldErrors = new ArrayList<>();
```